distributed/src/main/scala/asobu/distributed/gateway/Gateway.scala

```diff
@@ -45,20 +48,30 @@ class Gateway @Inject() (implicit system: ActorSystem, handlerBridgeProps: Handl
  * Eagerly start the Gateway
  */
 class GateWayModule extends Module {
+
   protected def defaultBridgeClass: Class[_ <: HandlerBridgeProps] = classOf[DefaultHandlerBridgeProps]
+  protected def defaultInterpreterClass: Class[_ <: Interpreter] = classOf[DisabledCustomExtractorInterpreter]
+
+  private class syntax[T: ClassTag](self: Option[Class[_ <: T]]) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Class name does not match the regular expression '^[A-Z][A-Za-z0-9]*$'](https://www.codacy.com/app/iHeartRadio/asobu/file/2681756948/issues/source?bid=3037259&fileBranchId=3335316#l55)